.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
   directories:
     - $HOME/.composer/cache
     - build/cache
+    - .tools
```

Comment:
What if we ever want to update the tools?

Reply:
Discussed in [this thread](https://github.com/infection/infection/pull/706#discussion_r294505032). It might be wiser to disable the caching of that directory meanwhile though so that the problem doesn't propagate to the CI yet

Reply:
IIRC the cache doesn't persist between PR's. 
And maybe we can make the cache 'depend' on the Makefile? like appveyor allows you to purge the cache if a specific file changes

Reply:
We can actually pull this off.

Reply:
I think if we employee the cache mechanism used in the other thread we are good, but Travis allows you to clear the cache if desired so.

The PR cache is not triggered until merged. Once merged, the following PRs will benefit from it (like for the Composer cache)